subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -187,66 +146,12 @@ Similarly, you can use a <<build_environment, project or system property>> to de
-
-[[publishing_maven:conditional_publishing]]
-=== Conditional publishing
-
-When you have defined multiple publications or repositories, you often want to control which publications are published to which repositories. For instance, consider the following sample that defines two publications and two repositories:
-
-++++
-<sample dir="maven-publish/conditional-publishing" id="multiplePublicationsAndRepositories" title="Adding multiple publications and repositories">
-    <sourcefile file="build.gradle" snippet="publishing"/>
-</sample>
-++++
-
-You may not want build users publishing both types of publications to both repositories, but the plugin automatically generates tasks for all possible combinations. So how do you stop someone from publishing the `binaryAndSources` publication to the `external` repository?
+For integration with a local Maven installation, it is sometimes useful to publish the module into the Maven local repository (typically at _$USER_HOME/.m2/repository), along with its POM file and other metadata. In Maven parlance, this is referred to as 'installing' the module.
 
-You can configure the tasks generated by the Maven Publish Plugin to be skipped based on certain criteria. The following sample demonstrates how to restrict the `binary` publication to the `external` repository and the `binaryAndSources` publication to the `internal` repository. In addition, it configures only `binaryAndSources` to be published to the local Maven repository.
-
-++++
-<sample dir="maven-publish/conditional-publishing" id="publishingMavenConditionally" title="Configuring which artifacts should be published to which repositories">
-    <sourcefile file="build.gradle" snippet="task-config"/>
-    <output args="publish publishToMavenLocal"/>
-</sample>
-++++
-
-Moreover, you may want to define your own shorthand tasks to fit your workflow. The following sample defines two tasks: `publishToExternalRepository` to publish all publications to the `external` repository and `publishForDevelopment` to publish all publications to the `internal` and the local Maven repositories:
-
-++++
-<sample dir="maven-publish/conditional-publishing" id="shorthandTasks" title="Defining your own shorthand tasks for publishing">
-    <sourcefile file="build.gradle" snippet="shorthand-tasks"/>
-</sample>
-++++
+The Maven Publish Plugin makes this easy to do by automatically creating a api:org.gradle.api.publish.maven.tasks.PublishToMavenLocal[] task for each api:org.gradle.api.publish.maven.MavenPublication[] in the `publishing.publications` container. The task name follows the pattern of `publish__PubName__PublicationToMavenLocal`. Each of these tasks is wired into the `publishToMavenLocal` aggregate task. You do not need to have `mavenLocal()` in your `publishing.repositories` section.
 
 [[publishing_maven:signing]]
 === Signing artifacts
```

Comment:
You mean the signing section? I wasn't sure about that, since it was only in the Maven chapter already. I'll take a look. Hopefully it works largely the same for Ivy publications too.

 - [ ] Review whether to move _Signing artifacts_ section to overview

Reply:
It works exactly the same.